corehq/apps/app_manager/tests/data/suite/reports_module_summary_detail.xml

```diff
@@ -31,7 +31,7 @@
         </header>
         <template>
           <text>
-            <xpath function="description"/>
+            <xpath function="report description"/>
```

Comment:
Super-picky, but this isn't legitimate xpath, it ought to be `function="'report description'"`

On that note, it's fairly confusing that people now have to enter `'My Report Description'` instead of just `My Report Description` in the UI. I don't think there's a good way around it without killing this new feature, but it's unfortunate.